coffee-rest/src/main/java/hu/icellmobilsoft/coffee/rest/log/RequestResponseLogger.java

```diff
@@ -377,17 +384,24 @@ protected String printResponseEntity(Object entity, Integer maxLogSize) {
      *            log prefix
      * @param maskingNeeded
      *            is masking sensitive data needed
+     * @param mediaType
+     *            media type of entity if relevant
+     * @return entity in string
      */
-    public String printEntity(Object entity, Integer maxLogSize, String prefix, boolean maskingNeeded) {
+    public String printEntity(Object entity, Integer maxLogSize, String prefix, boolean maskingNeeded, MediaType mediaType) {
         if (entity == null) {
             return null;
         }
         StringBuffer sb = new StringBuffer();
         String entityText = null;
         if (entity instanceof String) {
             entityText = (String) entity;
-        } else {
+        } else if (mediaType != null && StringUtils.containsIgnoreCase(mediaType.getSubtype(), "json")) {
```

Comment:
Lehet szebb lenne a litrerálok helyett a MediaType konstansait használni,

Reply:
Azért gondoltam hogy ez jobb lesz, mert a MediaType.equals metódusa a paraméterek egyezőségét is nézi, így viszont az nincs nézve. Jó példa erre a karakter kódolás. Ha UTF-8-as json van, akkor már nem jó a sima MediaType összehasonlítása.

De ha rosszul gondolom, természetesen javítom :)

Reply:
szerintem erre gondolt: `MediaType.APPLICATION_JSON.getSubtype()`

Reply:
Ez így érthető; esetleg az MedaType.isCompatible-el lehetne egy próbát tenni, az elvileg paraméterek nélkül nézi (sőt wildcard-ot is figyelembe veszi).

Reply:
atom+xml -re vagy text/xml-re mi legyen az entityText? Akkor is xml-ként kéne? Ha igen akkor az Imi megoldása lesz a jó, ha viszont nem kell akkor jó a Márké is.

Reply:
Erre a 2 xml-re jó az amit Imi írt, és azt használtam fel. Beleírtam a migrációs doksiba is.